cmd/admin/overrides_add.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"time"
+
+	"github.com/letsencrypt/boulder/config"
+	rl "github.com/letsencrypt/boulder/ratelimits"
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+	"google.golang.org/protobuf/types/known/durationpb"
+)
+
+type subcommandAddOverride struct {
+	limitEnum int64
+	regID     int64
+	domain    string
+	domains   string
+	ipAddr    string
+	count     int64
+	burst     int64
+	period    string
+	comment   string
+}
+
+func (*subcommandAddOverride) Desc() string {
+	return "Add or update a rate limit override. New overrides are enabled by default. Updates to existing overrides will not change the enabled state."
+}
+
+func (c *subcommandAddOverride) Flags(f *flag.FlagSet) {
+	f.Int64Var(&c.limitEnum, "limit", 0, "ratelimit enum (required)")
+	f.Int64Var(&c.regID, "regid", 0, "a single registration/account ID")
+	f.StringVar(&c.domain, "domain", "", "single domain (e.g. example.com)")
+	f.StringVar(&c.domains, "domains", "", "comma-separated list of FQDNs (e.g. example.com,www.example.com)")
+	f.StringVar(&c.ipAddr, "ip", "", "IPv4/IPv6 address")
+
+	f.Int64Var(&c.count, "count", 0, "allowed requests per period (required)")
+	f.Int64Var(&c.burst, "burst", 0, "burst size (required)")
+	f.StringVar(&c.period, "period", "", "period duration (e.g. 1h, 168h) (required)")
+	f.StringVar(&c.comment, "comment", "", "comment for the override (required)")
```

Comment:
I see the comment is optional in some structs in `ratelimits/limit.go`, but required here. I assume that's deliberate?

Reply:
It is deliberate here. Right now we do not require a comment when loading overrides from file, because not all overrides have a comment. But going forward all overrides coming from Zendesk will have a comment, and thus any being added/updated manually should also.